### ISSUE TITLE
vector_types::bindgen_test_layout_(u)long* fails on windows

### ISSUE THREAD

**Issue by termoshtt** [rust-cuda/cuda-sys#10]
```
failures:
    vector_types::bindgen_test_layout_long1
    vector_types::bindgen_test_layout_long2
    vector_types::bindgen_test_layout_long3
    vector_types::bindgen_test_layout_long4
    vector_types::bindgen_test_layout_ulong1
    vector_types::bindgen_test_layout_ulong2
    vector_types::bindgen_test_layout_ulong3
    vector_types::bindgen_test_layout_ulong4

test result: FAILED. 89 passed; 8 failed; 0 ignored; 0 measured; 0 filtered out
```

<details>

```
failures:

---- vector_types::bindgen_test_layout_long1 stdout ----
thread 'vector_types::bindgen_test_layout_long1' panicked at 'assertion failed: `(left == right)`
  left: `4`,
 right: `8`: Size of: long1', src\vector_types.rs:778:5
note: Run with `RUST_BACKTRACE=1` for a backtrace.

---- vector_types::bindgen_test_layout_long2 stdout ----
thread 'vector_types::bindgen_test_layout_long2' panicked at 'assertion failed: `(left == right)`
  left: `8`,
 right: `16`: Size of: long2', src\vector_types.rs:825:5

---- vector_types::bindgen_test_layout_long3 stdout ----
thread 'vector_types::bindgen_test_layout_long3' panicked at 'assertion failed: `(left == right)`
  left: `12`,
 right: `24`: Size of: long3', src\vector_types.rs:874:5

---- vector_types::bindgen_test_layout_long4 stdout ----
thread 'vector_types::bindgen_test_layout_long4' panicked at 'assertion failed: `(left == right)`
  left: `16`,
 right: `32`: Size of: long4', src\vector_types.rs:945:5

---- vector_types::bindgen_test_layout_ulong1 stdout ----
thread 'vector_types::bindgen_test_layout_ulong1' panicked at 'assertion failed: `(left == right)`
  left: `4`,
 right: `8`: Size of: ulong1', src\vector_types.rs:801:5

---- vector_types::bindgen_test_layout_ulong2 stdout ----
thread 'vector_types::bindgen_test_layout_ulong2' panicked at 'assertion failed: `(left == right)`
  left: `8`,
 right: `16`: Size of: ulong2', src\vector_types.rs:849:5

---- vector_types::bindgen_test_layout_ulong3 stdout ----
thread 'vector_types::bindgen_test_layout_ulong3' panicked at 'assertion failed: `(left == right)`
  left: `12`,
 right: `24`: Size of: ulong3', src\vector_types.rs:909:5

---- vector_types::bindgen_test_layout_ulong4 stdout ----
thread 'vector_types::bindgen_test_layout_ulong4' panicked at 'assertion failed: `(left == right)`
  left: `16`,
 right: `32`: Size of: ulong4', src\vector_types.rs:981:5

```

</details>

**Comment by AndrewGaspar** [comment]
This would presumably be fixed by #4

**Comment by termoshtt** [comment]
Resolved in #19
